src/ReleaseHistory.md

```diff
@@ -391,4 +391,5 @@
 * BUGFIX: `location.logicalLocation` convenience setter mishandles null. https://github.com/microsoft/sarif-sdk/issues/1514
 * BUGFIX: Upgrade schemas to latest version (remove `draft-04` from `$schema` property and change `id` to `$id`). This is necessary because the schemas use the `uri-reference` format, which was not defined in draft-04. https://github.com/microsoft/sarif-sdk/issues/1521
 * API BREAKING: The `Init` methods in the Autogenerated SARIF object model classes are now `protected virtual`. This enables derived classes to add additional properties without having to copy the entire code of the `Init` method.
-* BUGFIX: Transformation from SARIF 1.0 to 2.x throws ArgumentOutOfRangeException, if `result.locations` is an empty array. https://github.com/microsoft/sarif-sdk/issues/1526
\ No newline at end of file
+* BUGFIX: Transformation from SARIF 1.0 to 2.x throws ArgumentOutOfRangeException, if `result.locations` is an empty array. https://github.com/microsoft/sarif-sdk/issues/1526
+* BUGFIX: Improve Result Severity calculation for Fortify Converter based on MicroFocus feedback.
```

Comment:
>Severity [](start = 25, length = 8)

You could possibly justify capitalizing "Result" because it shows up in the SDK that way (although I personally wouldn't). But "Severity" isn't an SDK term (we use "Level"), so don't capitalize it.

Reply:
you are right! i will rephrase it to mention so

---
In reply to: [292522153](https://github.com/microsoft/sarif-sdk/pull/1528#discussion_r292522153) [](ancestors = 292522153)